ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/ShowCustomerCenter.kt

```diff
@@ -0,0 +1,33 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter
+
+import android.content.Context
+import android.content.Intent
+import androidx.activity.result.contract.ActivityResultContract
+
+/**
+ * Contract for launching the Customer Center.
+ *
+ * Usage:
+ * ```
+ * class MyActivity : ComponentActivity() {
+ *     private val customerCenter = registerForActivityResult(ShowCustomerCenter()) {
+ *         // Handle the dismissal
+ *     }
+ *
+ *     fun showCustomerCenter() {
+ *         customerCenter.launch()
+ *     }
+ * }
+ * ```
+ */
+class ShowCustomerCenter : ActivityResultContract<Unit, Unit>() {
+    override fun createIntent(context: Context, input: Unit): Intent {
+        return CustomerCenterActivity.createIntent(context).apply {
+            addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
```

Comment:
I'm always scared with these flags in Android TBH 😅 but `NEW_TASK` seems like it should work!

Reply:
https://github.com/RevenueCat/purchases-android/pull/2190 had to revert this lol

When opening the CustomerCenterActivity for result. I get a RESULT_CANCEL right away, even before the activity `onCreate`